other/s-z/update-image-tag/00-crb.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyverno:background-controller:update-image-tag
+  labels:
+    app.kubernetes.io/component: background-controller
+    app.kubernetes.io/instance: kyverno
+    app.kubernetes.io/part-of: kyverno
+rules:
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - update
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: update-image-tag-ns
```

Comment:
Please break this out into a separate resource manifest.

Reply:
Sure! Done.